src/db/cass/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM python:3.6
+
+WORKDIR /cass-app
+
+# Sets environment variables on the server
+ENV NGINX_RTMP_IP=127.0.0.1
+ENV NGINX_RTMP_PORT=19350
+ENV CASSANDRA_CLUSTER_IPS="192.168.1.15"
```

Comment:
I'm assuming this is just a placeholder ip right now?

Reply:
![aaaaa](https://user-images.githubusercontent.com/22970359/79267461-3094a280-7e67-11ea-84de-94538c632a8f.PNG)

Reply:
That's where your machine IP is on the local network. Try to use localhost.

Reply:
![yes](https://user-images.githubusercontent.com/22970359/79280499-7f016b80-7e7e-11ea-8ac2-f6cc55f132c8.PNG)

Reply:
![hello](https://user-images.githubusercontent.com/22970359/79280515-84f74c80-7e7e-11ea-9093-8cf979ccdbfd.PNG)

Reply:
I got these 2 errors about changing to localhost

Reply:
You need to run these in order: 
nginx-rtmp
cassandra
cass-app

Reply:
I take it back I forgot to mention that 127.0.0.1 points to the docker instance and not the host instance. So you need to route it via local WiFi network.